PROTOCOL.md

```diff
@@ -142,67 +139,6 @@ must match the `datacontentype` field.
 A Polyn client MUST support loading a JSON Schema document for each event. It MUST wrap the event
 schema into a valid CloudEvent JSON Schema, and publish that schema to the Schema Repository.
 
-#### Schema Repository
```

Comment:
It makes sense to remove this since it is no longer the job of the clients to publish the schemas, however I think it would be useful to indicate here the name of the KeyValue bucket and the naming convention of the keys so that client implementations know where to look for the schemas.